chapters/samplers.txt

```diff
@@ -856,3 +881,40 @@ include::{generated}/validity/protos/vkDestroySamplerYcbcrConversion.txt[]
 --
 
 endif::VK_VERSION_1_1,VK_KHR_sampler_ycbcr_conversion[]
+
+ifdef::VK_EXT_custom_border_color[]
+
+[open,refpage='VkSamplerCustomBorderColorCreateInfoEXT',desc='Structure specifying custom border color',type='structs']
+--
+
+Applications can provide a custom border color which is not listed under
+ename:VkBorderColor, this structure allows for specifying a custom color value
+to be added to the slink:VkSamplerCreateInfo::pname:pNext chain.
+
+The sname:VkSamplerCustomBorderColorCreateInfoEXT structure is defined as:
+
+include::{generated}/api/structs/VkSamplerCustomBorderColorCreateInfoEXT.txt[]
+
+  * pname:sType is the type of this structure.
+  * pname:pNext is `NULL` or a pointer to an extension-specific structure.
+  * pname:borderColor is a slink:VkClearColorValue representing the desired
+    custom sampler color.
+
+.Valid Usage
+****
+  * [[VUID-VkSamplerCustomBorderColorCreateInfo-maxUnique]]
+  The maximum number of custom border colors which can: be simultaneously
+  created on a device is implementation-dependent and specfied by the
+  <<limits-maxUniqueCustomBorderColors,maxUniqueCustomBorderColors>> member of
+  the slink:VkPhysicalDeviceCustomBorderColorPropertiesEXT structure. If
+  pname:maxUniqueCustomBorderColors is zero, the implementation supports an
+  unlimited number of custom border colors.
+  If pname:maxUniqueCustomBorderColors is exceeded, fname:vkCreateSampler will
+  return ename:VK_ERROR_TOO_MANY_OBJECTS
```

Comment:
If this is "valid but returns an error", then it shouldn't be a VUID. VUIDs specify cases that result in undefined behavior.